app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -140,72 +141,86 @@ class FollowUsers extends Component {
-                    </p>
-                  </div>
-                  <button
-                    data-testid="onboarding-user-following-status"
-                    type="button"
-                    className="user-following-status"
-                  >
-                    {selectedUsers.includes(user) ? 'Following' : 'Follow'}
-                  </button>
-                </button>
-              ))}
+      <Focustrap>
+        <div
+          data-testid="onboarding-follow-users"
+          className="onboarding-main crayons-modal"
+        >
+          <div
+            className="crayons-modal__box overflow-auto"
+            role="dialog"
+            aria-labelledby="title"
+            aria-describedby="subtitle"
+          >
+            <Navigation
+              prev={prev}
+              next={this.handleComplete}
+              canSkip={canSkip}
+              slidesCount={slidesCount}
+              currentSlideIndex={currentSlideIndex}
+            />
+            <div className="onboarding-content toggle-bottom">
+              <header className="onboarding-content-header">
+                <h1 id="title" className="title">
+                  Suggested people to follow
+                </h1>
+                <h2 id="subtitle" className="subtitle">
+                  Let&apos;s review a few things first
+                </h2>
+              </header>
+
+              <ul
+                data-testid="onboarding-users"
+                className="onboarding-modal-scroll-container"
+              >
+                {users.map((user) => (
+                  <li key={user.name}>
+                    <button
+                      data-testid="onboarding-user-button"
+                      type="button"
+                      onClick={() => this.handleClick(user)}
+                      onKeyDown={() => this.handleKeyDown(user)}
+                      className={
+                        selectedUsers.includes(user)
+                          ? 'user content-row selected'
+                          : 'user content-row unselected'
+                      }
+                    >
+                      <figure className="user-avatar-container">
+                        <img
+                          className="user-avatar"
+                          src={user.profile_image_url}
+                          alt="profile"
+                        />
+                      </figure>
+                      <div className="user-info">
+                        <h4 className="user-name">{user.name}</h4>
+                        <p className="user-summary">
+                          {he.unescape(user.summary || '')}
+                        </p>
+                      </div>
+                      <button
+                        data-testid="onboarding-user-following-status"
+                        type="button"
+                        className="user-following-status"
+                        aria-pressed={selectedUsers.includes(user)}
```

Comment:
Here would could add 

```jsx
aria-label={`${selectedUsers.includes(user) ? 'Unfollow' : 'Follow'} ${user.name}`}
```

This will tell the screen reader what to read.

Reply:
This would be to coincide with the `tabIndex` change.

Reply:
> Here would could add
> 
> ```js-jsx
> aria-label={`${selectedUsers.includes(user) ? 'Unfollow' : 'Follow'} ${user.name}`}
> ```
> 
> This will tell the screen reader what to read.

 But it would miss the description for each user

Reply:
I guess the three options are

- We remove something from the tabIndex be it the inner or outer button
- We accept multiple tabs is a thing that people have to deal with
- We wait for #10468 and use the useListNavigation hook to add j and k navigation support.